lib/rdoc/generator/json_index.rb

```diff
@@ -193,7 +193,7 @@ def generate_gzipped
         debug_msg "Writing gzipped file to %s" % outfile
 
         Zlib::GzipWriter.open(outfile) do |gz|
-          gz.mtime = File.mtime(dest)
+          gz.mtime = 1 # make output reproducible
```

Comment:
Please fix `FileUtils.install` with `:preserve` option in `RDoc::Generator::JsonIndex#generate` because the `mtime` is based on original `.js` file.

For your information, `FileUtils.install` is based on "install(1)" command.